CHANGES.rst

```diff
@@ -17,6 +17,9 @@ debops.apache v0.1.0 - unreleased
 Added
 ~~~~~
 
+- Added a apache__redirect_to_https to control the role's default behaviour for
+  redirecting to https. [muelli_]
```

Comment:
hrm. creating that entry seems to require a non trivial addition to debops-keyring, too.  Is the procedure defined anywhere?  Anyway, I've tried to create the necessary pull requests.

But there is no change the file here requires, right?